### PR TITLE
Fixes ayylmao tongues not allowing communication unless you're antag

### PR DESCRIPTION
**Pull request by Jujumatic**
Fixed a weird ayylmao check that made you have to be an antag to speak in the ayylmao hivemind chat. I've added a second channel that all chat will be redirected to, if the person speaking is not an abductor antag.

:cl: 
fix: Fixed ayylmao tongues not allowing you to talk unless you have antag status. Now there's one chat for abductors and another chat for everyone else.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
```
/obj/item/organ/tongue/abductor/TongueSpeech(var/message)
	//Hacks
	var/mob/living/carbon/human/user = usr
	var/rendered = "<span class='abductor'><b>[user.name]:</b> [message]</span>"
	log_talk(user,"ABDUCTOR:[key_name(user)] : [rendered]",LOGSAY)
	for(var/mob/living/carbon/human/H in GLOB.alive_mob_list)
		var/obj/item/organ/tongue/T = H.getorganslot(ORGAN_SLOT_TONGUE)
		if(!T || T.type != type)
			continue
		if(H.dna && H.dna.species.id == "abductor" && user.dna && user.dna.species.id == "abductor")
			var/datum/antagonist/abductor/A = user.mind.has_antag_datum(/datum/antagonist/abductor)
			if(!A || !(H.mind in A.team.members))
				continue
		to_chat(H, rendered)
	for(var/mob/M in GLOB.dead_mob_list)
		var/link = FOLLOW_LINK(M, user)
		to_chat(M, "[link] [rendered]")
	return ""
```
From what I can see of this code, it only restricts abductors talking to abductors based if they're on the same team or not. What might be better is to check for a speech modifier like ".a" for all or something. Abductor team chats shouldn't be compromised imo, although that for the headmins to decide.

**Comment by Jujumatic** [comment]
It'd open up new strategies where everyone mutates themselves into ayys to fuck with the abductors, stuff like that

**Comment by ghost** [comment]
yeah but that's like saying people should be able to access syndie radio to fuck up nukies
there's a good ingame way to handle it which is to steal one from a nuke op

i have to call into question the ability to compromise abductor teamchat like this

**Comment by Jujumatic** [comment]
fair enough - but it's not like it's easy to get the stuff to turn yourself into an ayy in the first place so it would be a rare occurrence

**Comment by ghost** [comment]
i don't think encouraging power gaming to be able to compromise secure chats is good
i think enabling communication between abductor teams is good

**Comment by Jujumatic** [comment]
yeah but unless the entire crew turns into abductors you can't communicate with the crew very well so i think it's balanced out. if the entire crew somehow manages to turn into abductors... well - i think this would be pretty fun

**Comment by praisenarsie** [comment]
do this upstream

**Comment by Jujumatic** [comment]
updated this and no i'm not doing this upstream as of yet because the setup is a fucking nightmare and i've already tried for 4 hours straight

**Comment by Jujumatic** [comment]
not actually sure how to set two teams and have only one of them being viewable depending on whether you're an actual abductor or crew

**Comment by Jujumatic** [comment]
Also changing this so there's no species check, if you give yourself an ayy tongue but you're not an ayy, you're basically a mute. which is stupid

**Comment by JohnGinnane** [comment]
If I got a tongue transplant with a chinese person, can I speak and understand chinese? :thinking: 

Joking aside, I like the idea of using an alien tongue to *somehow* speak and understand aliens and perhaps remove the ability to speak and understand humans

**Comment by Jujumatic** [comment]
i'll see if i can make having the ayy tongue as a human removes your ability to understand galactic common, but i'll do that later. also, testing this is really hard because i don't know how to force transform people into ayys

**Comment by JohnGinnane** [comment]
Right click > Antag (or Traitor) Panel > Make alien

Or try right click > Player Panel > Antag (or Traitor) Panel > Make alien

File: code/modules/surgery/organs/tongue.dm [hunk 1]
```diff
@@ -92,7 +92,7 @@
 			var/datum/antagonist/abductor/A = user.mind.has_antag_datum(/datum/antagonist/abductor)
 			if(!A || !(H.mind in A.team.members))
 				continue
-		to_chat(H, rendered)
+			to_chat(H, rendered)
```
Comment (GuyonBroadway) [inline code/modules/surgery/organs/tongue.dm]: add a comment after this that says //Hippie - desc of thing I did here and why

**Review by GuyonBroadway — DISMISSED** [review]

File: code/modules/surgery/organs/tongue.dm (hunk 1 above)
Comment (Jujumatic) [reply code/modules/surgery/organs/tongue.dm]: awe fuck i didn't mean to change this one, is it possible to revert this file?

**Comment by Jujumatic** [comment]
fixed

**Comment by JohnGinnane** [comment]
@Jujumatic is this ready and did you test it

@HippieStation/head can I get some input please

**Comment by Jujumatic** [comment]
i will test it once i get back, I've been spending all my time looking for a fix for that damned split personality stuff

**Comment by Jujumatic** [comment]
ok i tested this - it seems that the messages successfully send, when a non antag gets transformed into an abductor however the messages can only be viewed by deadchat for some reason - no idea why

**Comment by Jujumatic** [comment]
changed some stuff up, it needs testing and the abductor and other vars are unused according to the compiler. but anyway you should see what i'm trying to do. also don't mind the change i made to that one chem file, it's for testing purposes - it seems that people transformed into abductors via chemicals have something working differently to actual antag abductors

**Comment by Jujumatic** [comment]
This took me SO long to do, but it's completely done AND tested - well at least the crew team got tested. For some reason I couldn't force create antag abductors even through the admin menu, but codewise it's not much different from how the crew's abductor team works, it just has a check that worked in the initial abductor tongue code. plisx review and +999 this because it literally took me weeks to figure this stupid shit out

**Comment by earthmoonjune** [comment]
+1

**Review by praisenarsie — APPROVED** [review]

**Comment by praisenarsie** [comment]
if this breaks abductors i'm sueing

**Comment by Jujumatic** [comment]
no...... me!!!

**Pull request merged**